tracer/test/test-applications/integrations/Samples.XUnitTests/Samples.XUnitTests.csproj

```diff
@@ -24,6 +24,10 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="coverlet.collector" Version="6.0.0">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
```

Comment:
I wonder how this will break if we enable code coverage collection ourselves 😬 You can test in the CI if you're feeling brave...

Reply:
I'm not feeling brave enough 😄

Reply:
I see, making this a "platform team" issue, gotcha 😛